container_files/guac/guac.yaml

```diff
@@ -17,7 +17,7 @@ csub-addr: guac-collectsub:2782
 csub-listen-port: 2782
 
 # graphql
-gql-backend: inmem
+gql-backend: ent # ent|neo4j|inmem
```

Comment:
I think here we need to keep inmem as default for now?

Reply:
@ivanvanderbyl can you change this to be inmem by default